### PR TITLE
Firefox41

### PR DESCRIPTION
**Pull request by Guillaume28**
Problem about display in firefox 41 fixed.
SQL request when list is empty fixed.
#53 and #28 compare

### CODE REVIEW

File: service-monitoring/data.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ function loadPage()
             jQuery("#serviceMonitoringTable").empty().append(htmlData);
             var hostMonitoringTable = jQuery("#serviceMonitoringTable").find("img, style, script, link").load(function () {
                 var h = document.getElementById("serviceMonitoringTable").scrollHeight + 50;
-                parent.iResize(window.name, h);
+                ResizeFrame(window.name, h);
```
Comment (kduret) [inline service-monitoring/data.js]: Why use an other method ?

File: service-monitoring/data.js [hunk 2]
```diff
@@ -50,4 +50,12 @@ jQuery(function () {
...
     });
-});
\ No newline at end of file
+});
+
+function ResizeFrame(ifrm, height) {
+    if (height < 150) {
+        height = 150;
+    }
+    console.log(jQuery('[name="' + ifrm + '"]'));
```
Comment (kduret) [inline service-monitoring/data.js]: Remove console.log

File: service-monitoring/src/toolbar.php [hunk 3]
```diff
@@ -1,154 +1 @@
...
                                });
            } else {
                alert("<?php echo _('Please select one or more items'); ?>"); 
                return false;
            }
            $(".toolbar").val(0);
        }
    });
});
</script>
```
Comment (kduret) [inline service-monitoring/src/toolbar.php]: bad encode (^M)

File: service-monitoring/src/index.php [hunk 4]
```diff
@@ -229,15 +229,17 @@
...
         $Host[] = $row['host_id'];
     }
-    
-    $query = CentreonUtils::conditionBuilder($query, 
-      " s.service_id IN (
-            SELECT DISTINCT s.service_id FROM servicegroups sg, services_servicegroups sgm, 
-            services s, hosts h WHERE h.host_id = s.host_id AND s.host_id = sgm.host_id AND s.service_id = sgm.service_id 
+
+    if(count($Host) === 0){
+        $query = CentreonUtils::conditionBuilder($query,
```
Comment (loiclau) [inline service-monitoring/src/index.php]: style

File: service-monitoring/src/index.php [hunk 5]
```diff
@@ -229,15 +229,17 @@
     while ($row = $resultHost->fetchRow()) {
         $Host[] = $row['host_id'];
     }
-    
-    $query = CentreonUtils::conditionBuilder($query, 
-      " s.service_id IN (
-            SELECT DISTINCT s.service_id FROM servicegroups sg, services_servicegroups sgm, 
-            services s, hosts h WHERE h.host_id = s.host_id AND s.host_id = sgm.host_id AND s.service_id = sgm.service_id 
+
+    if(count($Host) === 0){
```
Comment (loiclau) [inline service-monitoring/src/index.php]: good condition? implode on host line 240

**Comment by kduret** [comment]
not relevant anymore

**Pull request closed without merging**